hooks/post_gen_project.py

```diff
@@ -0,0 +1,24 @@
+import re
+from packaging.version import parse
+
+def update_vbu_pin():
```

Comment:
this is totally new. cookiecutter supports three types of hooks: pre_prompt (which is what we already use for prompting the user for versions), pre_get_project, and post_gen_project. In this case, we leverage the latter to overwrite the vbu pin in the setup.py to include the upper bound rather than the more standard PACKAGE>=VERSION